specs/fizzbuzz_spec.rb

```diff
@@ -0,0 +1,32 @@
+# write only unit tests for method fizzbuzz in class Solver
+
+require 'rspec'
+require '../solver'
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/229285751-e8743250-7f32-4d36-92ae-b559c34d4818.png)

- [ ] Kindly use required_relative to import the solver class because now `rspec` cannot find any solver file :+1: and I am getting the error showed above :arrow_up:

:green_circle: **After adding require_relative**

```ruby
require 'rspec'
require_relative '../solver'
```